internal/generator/vector/output/loki/with_headers.toml

```diff
@@ -0,0 +1,47 @@
+[transforms.loki_receiver_remap]
+type = "remap"
+inputs = ["application"]
+source = '''
+  del(.tag)
+'''
+
+[transforms.loki_receiver_remap_label]
+type = "remap"
+inputs = ["loki_receiver_remap"]
+source = '''
+if !exists(.kubernetes.namespace_name) {
+  .kubernetes.namespace_name = ""
+}
+if !exists(.kubernetes.pod_name) {
+  .kubernetes.pod_name = ""
+}
+if !exists(.kubernetes.container_name) {
+  .kubernetes.container_name = ""
+}
+'''
+
+[sinks.loki_receiver]
+type = "loki"
+inputs = ["loki_receiver_remap_label"]
+endpoint = "https://logs-us-west1.grafana.net"
+out_of_order_action = "accept"
+healthcheck.enabled = false
+
+[sinks.loki_receiver.encoding]
+codec = "json"
+except_fields = ["_internal"]
+
+[sinks.loki_receiver.request]
```

Comment:
@AndrewChubatiuk  Thanks for your contribution! Just a heads-up: according to the official docs, the Loki sink’s request block doesn’t support a headers field. https://vector.dev/docs/reference/configuration/sinks/loki/#request

Reply:
right, thanks, removed them for loki output